common/cliutils/utils.go

```diff
@@ -251,3 +253,9 @@ func FixWinPathsForFileSystemSourcedCmds(uploadSpec *spec.SpecFiles, specFlag, e
 		}
 	}
 }
+
+func PlatformToLifecycleUrls(lcDetails *coreConfig.ServerDetails) {
```

Comment:
This method isn't really a common util, it can probably be based in a lifecycle package. Maybe even in jfrog-cli-artifactory.

Reply:
agree moved to `jfrog-cli-artifactory`

Reply:
Looks like this is still here, did it move?

Reply:
It didn't move earlier, updated now